tests/lib/reset.sh

```diff
@@ -138,7 +138,15 @@ reset_all_snap() {
     done
     # remove all base/os snaps at the end
     if [ -n "$remove_bases" ]; then
-        snap remove "$remove_bases"
+        for base in $remove_bases; do
+            snap remove "$base"
+            if [ -d "$SNAP_MOUNT_DIR/$base" ]; then
+                echo "Error removing base snap $base"
```

Comment:
Maybe a message like: `"Error: removing base $base has unexpected leftover dir $SNAP_MOUNT_DIR/$base"` or something like this? Because at this point "snap remove $base" did not error (or we would not have reached this line) but for unknonwn reasons there is a leftover dir that we did not expect(?).

Reply:
Thanks for the review and the suggestion. Change done.